image/manifest.go

```diff
@@ -75,8 +75,15 @@ func (m *manifest) validate(w walker) error {
 		return errors.Wrap(err, "config validation failed")
 	}
 
+	validLayerMediaTypes := []string{
```

Comment:
Is there a reason to make this private?  If a third party wants to support an extension layer type (which we [explicitly allow][1]), that would give them a way to do it.

[1]: https://github.com/opencontainers/image-spec/blame/v1.0.0-rc4/manifest.md#L62